StripeFinancialConnections/StripeFinancialConnections/Source/Native/AccountPicker/AccountPickerFooterView.swift

```diff
@@ -74,32 +78,29 @@ final class AccountPickerFooterView: UIView {
     func didSelectAccounts(count numberOfAccountsSelected: Int) {
         linkAccountsButton.isEnabled = (numberOfAccountsSelected > 0)
 
-        if institutionHasAccountPicker {
-            linkAccountsButton.title = STPLocalizedString(
-                "Confirm",
-                "A button that allows users to confirm the process of saving their bank accounts for future payments. This button appears in a screen that allows users to select which bank accounts they want to use to pay for something."
-            )
-        } else {
-            let singleAccountButtonTitle = STPLocalizedString(
-                "Link account",
-                "A button that allows users to confirm the process of saving their bank account for future payments. This button appears in a screen that allows users to select which bank accounts they want to use to pay for something."
-            )
-            let multipleAccountButtonTitle = STPLocalizedString(
-                "Link accounts",
-                "A button that allows users to confirm the process of saving their bank accounts for future payments. This button appears in a screen that allows users to select which bank accounts they want to use to pay for something."
-            )
+        let singleAccountButtonTitle = STPLocalizedString(
```

Comment:
nit: we started to move newly added strings into`String+Localized.swift` for better readability, seems there's one for FC too, are you planning to add them there?

Reply:
FC right now is excluded from localization (part because we sometimes return strings from backend, so it wouldn't make sense to localize client, and then have non-localized strings from backend), so all of this is mostly to avoid future extra work for when we do plan to localize

I think FC was forced to create `String+Localized.swift` when localization WAS turned on because the script Stripe has blocked duplicate strings (even though that should actually not be a requirement - it can be handled automatically). So the file was used to solve that problem: reuse duplicate strings.

The pattern is debatable, but I always err on the side of simplicity: I _try_ to _only_ create abstractions when absolutely necessary (though I am definitely not perfect). So I try to only create extra files, extra abstractions, when I am "forced" to. This minimizes the chance of over-engineering. I can't claim that `String.Localized.linked_accounts` is worse, but I can't confidently claim that its better. It's completely debatable. As a result, the only thing I can rely on is principles: do the simplest thing possible and no more.

Reply:
I'm just used to all the strings defined in a file as android forces you to do so, but your concern is valid too

Reply:
ahh yea thats a good point on Android

iOS localization is a little odd but there's an Apple script that looks through _all_ of the codebase for a specific pattern and then it creates a file to send to localization 

this process is a little magical but I like it from the perspective is that, in an optimal case, localization should be as easy as just typing a string in-line

its not as magic as typing a string in-line, part because creating a comment for localizers is helpful, but it comes a little closer than putting the strings in specific files